astroid/brain/brain_io.py

```diff
@@ -35,9 +36,37 @@ def _transform_buffered(node):
     return _generic_io_transform(node, name='raw', cls=FileIO)
 
 
+def _enter_returns_self(node):
```

Comment:
It's duplicated here, because all brain modules weren't importing anything outside of `astroid` root.